<sdf version="1.4">
  <!--Generated by blend2world.py-->
  <world name="arena_8walls_blocks_ramps">
    <scene>
      <ambient>0.5 0.5 0.5 1</ambient>
      <background>0.5 0.5 0.5 1</background>
      <shadows>false</shadows>
    </scene>
    <physics type="ode">
      <gravity>0 0 -9.81</gravity>
      <ode>
        <solver>
          <type>quick</type>
          <iters>1000</iters>
          <sor>1.3</sor>
        </solver>
        <constraints>
          <cfm>0.0</cfm>
          <erp>0.2</erp>
          <contact_max_correcting_vel>100.0</contact_max_correcting_vel>
          <contact_surface_layer>0.001</contact_surface_layer>
        </constraints>
      </ode>
      <real_time_update_rate>120</real_time_update_rate>
      <real_time_factor>1</real_time_factor>
      <max_step_size>0.008333</max_step_size>
    </physics>
    <light name="directional_light_1" type="directional">
      <cast_shadows>false</cast_shadows>
      <pose>0 0 30 0.1 0.1 0</pose>
      <diffuse>1.0 1.0 1.0 1</diffuse>
      <specular>.1 .1 .1 1</specular>
      <attenuation>
        <range>300</range>
      </attenuation>
      <direction>0.1 0.1 -1</direction>
    </light>
    <include>
      <uri>model://ground_plane</uri>
    </include>
    <light name="directional_light.-10.0_-12.0" type="sun">
      <pose>-10.0 -12.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <light name="directional_light.-10.0_0.0" type="sun">
      <pose>-10.0 0.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <light name="directional_light.-10.0_12.0" type="sun">
      <pose>-10.0 12.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <light name="directional_light.0.0_-12.0" type="sun">
      <pose>0.0 -12.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <light name="directional_light.0.0_0.0" type="sun">
      <pose>0.0 0.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <light name="directional_light.0.0_12.0" type="sun">
      <pose>0.0 12.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <light name="directional_light.10.0_-12.0" type="sun">
      <pose>10.0 -12.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <light name="directional_light.10.0_0.0" type="sun">
      <pose>10.0 0.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <light name="directional_light.10.0_12.0" type="sun">
      <pose>10.0 12.0 8  0 0 0</pose>
      <cast_shadows>1</cast_shadows>
      <diffuse>0.5 0.5 0.5 1</diffuse>
      <specular>0.1 0.1 0.1 1</specular>
      <attenuation>
        <range>20</range>
        <constant>0.5</constant>
        <linear>0.01</linear>
        <quadratic>0.00</quadratic>
      </attenuation>
      <direction>0.1 0.1 -1.0</direction>
    </light>
    <include>
      <name>cube_20x20</name>
      <pose>-0.800000011920929 0.5099999904632568 0.10000000149011612  0.0 -0.0 3.141592502593994</pose>
      <uri>model://cube_20x20</uri>
    </include>
    <include>
      <name>cube_20x20.001</name>
      <pose>-1.0 0.3100000023841858 0.10000000149011612  0.0 -0.0 3.141592502593994</pose>
      <uri>model://cube_20x20</uri>
    </include>
    <include>
      <name>cube_20x20.002</name>
      <pose>-1.2000000476837158 0.5099999904632568 0.10000000149011612  0.0 -0.0 3.141592502593994</pose>
      <uri>model://cube_20x20</uri>
    </include>
    <include>
      <name>cube_20x20.003</name>
      <pose>-0.6000000238418579 0.5099999904632568 0.10000000149011612  0.0 -0.0 3.141592502593994</pose>
      <uri>model://cube_20x20</uri>
    </include>
    <include>
      <name>cube_20x20.004</name>
      <pose>-0.6000000238418579 0.10999999940395355 0.10000000149011612  0.0 -0.0 3.141592502593994</pose>
      <uri>model://cube_20x20</uri>
    </include>
    <include>
      <name>cube_20x30</name>
      <pose>-0.800000011920929 0.10999999940395355 0.15000000596046448  0.0 -0.0 3.141592502593994</pose>
      <uri>model://cube_20x30</uri>
    </include>
    <include>
      <name>cube_20x50</name>
      <pose>-0.800000011920929 0.3100000023841858 0.25  0.0 -0.0 3.141592502593994</pose>
      <uri>model://cube_20x50</uri>
    </include>
    <include>
      <name>ramp</name>
      <pose>1.2565653324127197 2.0179603099823 0.15000000596046448  0.0 0.0 0.0</pose>
      <uri>model://ramp</uri>
    </include>
    <include>
      <name>ramp.001</name>
      <pose>0.7436184883117676 2.0129599571228027 0.15000000596046448  0.0 -0.0 3.141592502593994</pose>
      <uri>model://ramp</uri>
    </include>
    <include>
      <name>ramp.002</name>
      <pose>-1.2133511304855347 1.7248939275741577 0.15000000596046448  0.0 -0.0 4.71238899230957</pose>
      <uri>model://ramp</uri>
    </include>
    <include>
      <name>ramp.004</name>
      <pose>-0.9832800030708313 2.2666399478912354 0.15000000596046448  0.0 -0.0 7.853981971740723</pose>
      <uri>model://ramp</uri>
    </include>
    <include>
      <name>wall_2x1</name>
      <pose>-1.0 0.0 0.5130000114440918  0.0 0.0 0.0</pose>
      <uri>model://wall_2x1</uri>
    </include>
    <include>
      <name>wall_2x1.001</name>
      <pose>1.0 0.0 0.5130000114440918  0.0 0.0 0.0</pose>
      <uri>model://wall_2x1</uri>
    </include>
    <include>
      <name>wall_2x1.002</name>
      <pose>0.0 1.0 0.5130000114440918  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_2x1</uri>
    </include>
    <include>
      <name>wall_2x1.003</name>
      <pose>0.0 3.0 0.5130000114440918  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_2x1</uri>
    </include>
    <include>
      <name>wall_2x1.004</name>
      <pose>-2.0 3.0 0.5130000114440918  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_2x1</uri>
    </include>
    <include>
      <name>wall_2x1.005</name>
      <pose>2.0 1.0 0.5130000114440918  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_2x1</uri>
    </include>
    <include>
      <name>wall_2x1.006</name>
      <pose>2.0 3.0 0.5130000114440918  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_2x1</uri>
    </include>
    <include>
      <name>wall_2x1.007</name>
      <pose>-1.0 2.0 0.5130000114440918  0.0 0.0 0.0</pose>
      <uri>model://wall_2x1</uri>
    </include>
    <include>
      <name>wall_stand</name>
      <pose>0.0 0.19949999451637268 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.001</name>
      <pose>0.19949999451637268 0.0 0.03449999913573265  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.002</name>
      <pose>-0.19949999451637268 0.0 0.03449999913573265  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.003</name>
      <pose>1.8005000352859497 0.0 0.03449999913573265  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.004</name>
      <pose>-1.8005000352859497 0.0 0.03449999913573265  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.005</name>
      <pose>0.0 2.19950008392334 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.006</name>
      <pose>2.0 2.19950008392334 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.007</name>
      <pose>-2.0 2.19950008392334 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.008</name>
      <pose>-0.19949999451637268 2.0 0.03449999913573265  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.009</name>
      <pose>0.0 1.8005001544952393 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.010</name>
      <pose>2.0 3.8005001544952393 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.011</name>
      <pose>-2.0 3.8005001544952393 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.012</name>
      <pose>2.0 1.8005001544952393 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.014</name>
      <pose>2.0 0.19950008392333984 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.015</name>
      <pose>-1.8005000352859497 2.0 0.03449999913573265  0.0 -0.0 1.570796251296997</pose>
      <uri>model://wall_stand</uri>
    </include>
    <include>
      <name>wall_stand.016</name>
      <pose>0.0 3.8005001544952393 0.03449999913573265  0.0 0.0 0.0</pose>
      <uri>model://wall_stand</uri>
    </include>
  </world>
</sdf>
